bedrock/mozorg/templates/mozorg/home/includes/fundraising/takeover.html

```diff
@@ -0,0 +1,48 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+ <article class="fundraising-takeover">
+   <div class="header-strip">
+     <p><a href="{{ url('mozorg.home') }}" class="more">{{ _('Continue to mozilla.org') }}</a></p>
+   </div>
+   <section class="body">
+     <div class="content-container">
+       <header>
+         <h1>{{ high_res_img('home/takeover/wordmark.png', {'alt': 'Mozilla', 'width': '348', 'height': '92'}) }}</h1>
+         <h2>{{ _('Proudly non-profit. <br>Donate before Dec 31.') }}</h2>
+       </header>
+       <form id="fundraiser-form" method="get" action="{{ donate_url('mozillaorg_takeover') }}#amount-25">
+         <fieldset>
+           <label for="donate100">
+             <input type="radio" value="100" id="donate100" name="amount"> {{ _('$100') }}
+           </label>
+
+           <label for="donate50">
+             <input type="radio" value="50" id="donate50" name="amount"> {{ _('$50') }}
+           </label>
+
+           <label for="donate25">
+             <input type="radio" value="25" id="donate25" name="amount" checked> {{ _('$25') }}
+           </label>
+
+           <label for="donate15">
+             <input type="radio" value="15" id="donate15" name="amount"> {{ _('$15') }}
+           </label>
+
+           <label for="donate-custom">{{ _('Enter your own amount') }}</label>
+           <input type="text" name="amount-custom" id="donate-custom" />
+
+           <label for="monthly">
+             <input type="radio" value="monthly" id="monthly" name="donation-type"> {{ _('Monthly') }}
+           </label>
+
+           <label for="onetime">
+             <input type="radio" value="onetime" id="onetime" name="donation-type"> {{ _('Onetime') }}
```

Comment:
I know it's "One time" in https://donate.mozilla.org/en-US/ but "One-time" in the last design I see. Not confusing at all...